tools/generators/xcschemes/src/Generator/CreateCustomSchemeInfos.swift

```diff
@@ -591,6 +603,17 @@ set
 
         let useRunArgsAndEnv =
             try consumeArg("test-use-run-args-and-env", as: Bool.self, in: url)
+
+        let enableMainThreadChecker = try consumeArg(
+            "test-enable-main-thread-checker",
+            as: Bool.self,
+            in: url
+        )
+        let enableThreadPerformanceChecker = try consumeArg(
+            "test-enable-performance-anti-pattern-checker",
+            as: Bool.self,
+            in: url
+        )
```

Comment:
Since these arguments come after the sanitizers in `xcschemes_execution.bzl`, these should be later. I feel this should have been caught by a test 🤔.

Reply:
Indeed, I think they passed because they were the same values...